src/main/java/fi/foyt/foursquare/api/FoursquareApi.java

```diff
@@ -75,6 +75,12 @@
   private static final String DEFAULT_VERSION = "20140131";
 
   /**
+   * CDI eyes only
+   * @deprecated CDI eyes only
+   */
+  public FoursquareApi() {}
```

Comment:
About the deprecated, we are using this pattern in some projects like Apache HttpClient, Jedis, VRaptor, etc.
e.g: https://github.com/apache/httpclient/pull/27/files